app/src/main/java/org/fossasia/phimpme/base/BaseActivity.java

```diff
@@ -77,7 +77,7 @@ protected void onResume() {
     private void presentShowcaseSequence() {
 
         ShowcaseConfig config = new ShowcaseConfig();
-        config.setDelay(4000); // half second between each showcase view
+        config.setDelay(500); // half second between each showcase view
```

Comment:
Looking at the comment I can see a delay of half second. But this is 5 seconds. 50 should be used here right?

Reply:
@abishekvashok No the setDelay() method takes long integer in form of milliseconds. 500 is for 0.5 milliseconds.

Reply:
Sorry I forgot that 1000 milliseconds formed a second!

Reply:
it's Ok.